### ISSUE TITLE
Open editors: group is not announced

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#94120]
Steps to Reproduce:

1. enable VoiceOver
2. have multiple editor groups
3. click on entries in open editors view

=> 🐛 the output is weird to say the least. I think sometimes no group info is read, or if, it is wrong.

**Comment by bpasero** [comment]
Each entry seems to have a "Close Editor" when announced too...

**Comment by isidorn** [comment]
Open editors view definetnly needs some accessibility love.
Assigning to April to look into it then.

**Comment by isidorn** [comment]
Fixed with the use of accessibilityProvider in open editros view.
